sql-to-dbsp-compiler/SQL-compiler/src/main/java/org/dbsp/sqlCompiler/compiler/backend/DBSPCompiler.java

```diff
@@ -229,21 +229,25 @@ private void compileInternal(String statements, boolean many, @Nullable String c
         } catch (SqlParseException e) {
             this.messages.reportError(e);
             if (this.options.optimizerOptions.throwOnError) {
+                System.err.println(this.messages.toString());
                 throw new RuntimeException(e);
             }
         } catch (CalciteContextException e) {
             this.messages.reportError(e);
             if (this.options.optimizerOptions.throwOnError) {
+                System.err.println(this.messages.toString());
```

Comment:
Should we be logging these instead of reporting to stderr?

Reply:
This is actually just for debugging. The 'throwOnError' flag is only designed for development.
The effect of this is to print in the IDE console some much nicer errors in addition to a stack trace:
```
(no input file): Error parsing SQL
(no input file):23:17: error: Error parsing SQL: Encountered "," at line 23, column 17.
Was expecting:
    "IN" ...
    
   23|        FOR name, age) IN (('John', 30) AS c1, ('Mike', 40) AS c2)
                      ^^
```